src/v/datalake/translation/state_machine.cc

```diff
@@ -59,8 +62,14 @@ ss::future<> translation_stm::do_apply(const model::record_batch& batch) {
           record.release_value());
         vlog(
           _log.trace,
-          "updating highest translated offset to {}",
-          value.highest_translated_offset);
+          "updating highest translated offset to {}[]",
```

Comment:
missing placeholder for timestamp

Reply:
removed this log

Reply:
timestamp semantics changed, removed from log line